ps06/CS2_2024_unicycle_discrete_stability.ipynb

```diff
@@ -0,0 +1,1323 @@
+{
```

Comment:
<p>maybe a hint that there is a function in ct for implementing the step response.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/idsc-frazzoli/CS2-2024-notebooks/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>